.github/workflows/tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
+    name: Node ${{ matrix.node }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Install Dependencies
+      run: npm ci && npm ls yargs && npm list -g --depth 0
```

Comment:
was this `npm ls yargs && npm list -g --depth 0` needed for the yargs issue with GH actions? can you point me to the link where the issue with the yargs version is explained?

Reply:
Ah yes I need to remove that 🤦🏼 https://github.com/yargs/yargs/pull/1317